_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -0,0 +1,288 @@
+---
+id: b8bb68
+name: Text spacing in `style` attributes is not `!important`
+rule_type: atomic
+description: |
+  This rule checks that text spacing set in the `style` attribute is below permissible minimum value or not set to `!important`
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
```

Comment:
Agree. This example should pass the SC as the line-height is above the threshold and so "!important" doesn't matter.

If "!important" were included in the applicability, then this would become an inapplicable example.